lib/rides/pages/ride_detail_page.dart

```diff
@@ -78,6 +80,9 @@ class _RideDetailPageState extends State<RideDetailPage> {
       int id = _ride?.id ?? widget.id!;
       Map<String, dynamic> data =
           await SupabaseManager.supabaseClient.from('rides').select(_rideQuery).eq('id', id).single();
+      Map<String, dynamic> dataChat =
```

Comment:
the stack trace does not show it clearly, but I get this error:

> [VERBOSE-2:dart_vm_initializer.cc(41)] Unhandled Exception: PostgrestException(message: JSON object requested, multiple (or no) rows returned, code: PGRST116, details: Results contain 0 rows, application/vnd.pgrst.object+json requires 1 row, hint: null)
> #0      PostgrestBuilder._parseResponse (package:postgrest/src/postgrest_builder.dart:323:7)
> #1      PostgrestBuilder._execute (package:postgrest/src/postgrest_builder.dart:198:14)
> <asynchronous suspension>
> #2      PostgrestBuilder.then (package:postgrest/src/postgrest_builder.dart:400:24)
> <asynchronous suspension

Reply:
I think that it could not find the chat there -can you reproduce that or have any idea why?

Reply:
that should not happen, since there always will be a chat at the moment because it gets created with the ride

Reply:
can't reproduce that error but i turned RLS of i think there's a mistake somewhere in the policies i'll check it out tomorrow

Reply:
could not reproduce it anymore